go/consensus/api/api.go

```diff
@@ -57,6 +57,9 @@ type ClientBackend interface {
 	// in the future).
 	WaitEpoch(ctx context.Context, epoch epochtime.EpochTime) error
 
+	// GetEpoch returns the current epoch.
+	GetEpoch(ctx context.Context, height int64) (epochtime.EpochTime, error)
```

Comment:
Add test for the new method (see `go/consensus/tests`).

Reply:
done